src/pages/resultsView/settings/SettingsMenu.tsx

```diff
@@ -111,122 +112,136 @@ export default class SettingsMenu extends React.Component<
                 <h5 style={{ marginTop: 'auto', marginBottom: 'auto' }}>
                     Annotate Data
                 </h5>
-                <InfoIcon
-                    divStyle={{ display: 'inline-block', marginLeft: 6 }}
-                    style={{ color: 'rgb(54, 134, 194)' }}
-                    tooltip={
-                        <span>
-                            Putative driver vs VUS setings apply to every tab
-                            except{' '}
-                            <BoldedSpanList
-                                words={['Co-expression', 'CN Segments']}
-                            />
-                        </span>
-                    }
-                />
+                {!this.props.comparisonView && (
```

Comment:
Agreed.  Unless someone asked for this specifically, I think this can be handled outside, next to the button/text that opens the menu.  In fact, since the button is quite specific to the tab (not in header) it may be implicit that it only applies to this tab.

Reply:
SettingsMenu will have a prop titleComponent
you just pass this jsx